packages/@o3r/testing/schematics/ng-add/index.ts

```diff
@@ -114,7 +114,8 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
           const jestConfigFilesForProject = () => mergeWith(apply(url('./templates/project'), [
             template({
               ...options,
-              rootRelativePath: path.posix.relative(workingDirectory, tree.root.path.replace(/^\//, './'))
+              rootRelativePath: path.posix.relative(workingDirectory, tree.root.path.replace(/^\//, './')),
+              isAngularSetup: tree.exists('/angular.json')
```

Comment:
I don't think this condition is accurate
This condition will almost always be true, but we only want to set the angular setup on jest.config if it's needed (not the case for the sdk for example)
Maybe we could check if @angular is present in the dependencies of the project ?

Reply:
sdk generation is a special case. Today it is generated with its own testing setup, nothing to do with `o3r/testing` package. 
For a monorepo case we have a bit more complex story which is described in this issue https://github.com/AmadeusITGroup/otter/issues/882 to align the generation of the possible projects (app/lib/sdk)